esmvaltool/cmor/_fixes/OBS/NCEP_NCAR.py

```diff
@@ -0,0 +1,15 @@
+"""Fixes for NCEP-NCAR"""
+
+from iris import Constraint
+
+from ..fix import Fix
+
+
+class zg(Fix):
+    """Class to fix clisccp"""
+
+    def fix_raw_cubes(self, cubes):
+        zg_constraint = Constraint(cube_func=(lambda c: c.var_name == 'zg'))
+        zg_cube = cubes.extract(zg_constraint)[0]
+        zg_cube.standard_name = 'geopotential_height'
```

Comment:
We are probably talking about two different datasets, since in `NCEP` this fix is not required.

Reply:
Is just the data I have with this name in PRIMAVERA. I will remove this fix from here and keep a PRIMAVERA_OBS project in PRIMAVERA branch.

At some point I will have to look in a way to describe custom fixes for platforms in which we can not rely in people been able to process observations according to our standards and also to allow adding fixes without having to touch the code of the tool

Reply:
Once #232 is solved, we should be able to maintain the reformat scripts for the observations as part of the distribution and generate data accordingly.